etc/cn-standalone-test.toml

```diff
@@ -12,9 +12,22 @@ service-addresses = [
     "2"
 ]
 
+[cn]
+uuid = "5849c262-2444-11ed-ba8c-d6aee46d73fa"
+
 [cn.Engine]
 type = "tae"
 
+[[fileservice]]
+# s3 fileservice instance, used to store data.
+name = "s3"
+# use disk as fileservice backend.
+backend = "DISK"
+# set the directory used by DISK backend. There must has a file named "thisisalocalfileservicedir"
+# in the data dir
+data-dir = "store"
+
```

Comment:
All CN and DN must contain two file services, 'local' and 's3'
for testing purposes, the backend should be 'MEM'
If you need an ETLFileService, just add another fs named 'etl' with 'DISK-ETL' or 'S3' backend